### PR TITLE
Add Casper network namespace (CAIP-2, CAIP-10)

### PR DESCRIPTION
**Pull request by davidatwhiletrue**
Adding Casper network namespace with CAIP-2 and CAIP-10 documents.

### CODE REVIEW

File: casper/caip2.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* Testnet: http://52.35.59.254:7777/rpc
+
+
+## References
+
+- [Casper Docs site][]
+- [CAIP-2][]
+
+[CAIP-2]: https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md
+[Casper Docs site]: https://docs.casper.network/
```
Comment (bumblefudge) [inline casper/caip2.md:66]: [Chainspec]: https://docs.casper.network/operators/setup-network/chain-spec/

Comment (bumblefudge) [inline casper/caip2.md:66]:
```suggestion
[Casper Docs site]: https://docs.casper.network/
[Chainspec]: https://docs.casper.network/operators/setup-network/chain-spec/
```

File: casper/caip2.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+For testing purposes, the Casper Association maintains the following RPC endpoints:
+
+* Casper Mainnet: http://34.224.191.55:7777/rpc
+* Testnet: http://52.35.59.254:7777/rpc
+
+
+## References
+
+- [Casper Docs site][]
```
Comment (bumblefudge) [inline casper/caip2.md:62]:
```suggestion
- [Casper Docs site][]
- [Casper Chain Specification][Chainspec]
```

File: casper/caip2.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+The Chain ID is a human-readable identifier for a Casper network blockchain. It should not be confused with the `genesis_hash`, which is the true and unique identifier of a chain.
+
+## Syntax
+
+It consists of the prefix `casper:` followed by the chainspec name of that network. Such name is a string that permits to easily identify the network and protect from replay attacks.
+
+### Resolution Method
+
+To resolve the chainspec name for a Casper network chain send an RPC request with method `info_get_status` to a Casper node in that network. The result object in the RPC response will contain the field `chainspec_name` with the chainspec name of that network.
```
Comment (bumblefudge) [inline casper/caip2.md:25]:
```suggestion
To resolve the [chainspec][] name for a given Casper network chain, send an RPC request with method `info_get_status` to a Casper node in that network. The result object in the RPC response will contain the property `chainspec_name` containing the [chainspec][] name corresponding to that network.
```

**Review by bumblefudge — APPROVED** [review]
LTGM, thanks!

If you want to make custom RPC methods available to WC wallets that may not necessarily implement the rest of your protocol, you might also want to open a separate PR with a CAIP-27 profile explaining how to send a dapp one-off RPC calls through wallet connect.

**Review by ganchoradkov — APPROVED** [review]

**Pull request merged**